westcoast/src/main/java/frc/motorFactory/MotorFactoryHybrid.java

```diff
@@ -5,9 +5,9 @@
 import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
 
 public class MotorFactoryHybrid implements MotorFactory {
-    public SpeedController create(int channelMain, int channel1, int channel2) {
-        WPI_TalonSRX motorMain = new WPI_TalonSRX(channelMain);
+    public SpeedController create(int channel1, int channelMain, int channel2) {
         WPI_VictorSPX motor1 = new WPI_VictorSPX(channel1);
+        WPI_TalonSRX motorMain = new WPI_TalonSRX(channelMain);
```

Comment:
I don't think it's a good idea to change the order of the parameters as they are not clearly typed, but mostly is a style issue.
Except for the controllers instantiation. I will keep the same types together and this is just part style, there is another technical reason (icache friendly but it is a long shot, so in the end is just style XD).

Anyway changing in this implementation means you have to change in `MotorFactory` declaration and in each other `MotorFactory` implementation in the project to keep them working (I think we have 3 in total).

Reply:
alright, rolled back